app/controllers/CountryOfConsumptionFromNiController.scala

```diff
@@ -35,12 +36,13 @@ class CountryOfConsumptionFromNiController @Inject()(
                                         view: CountryOfConsumptionFromNiView
                                     )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
 
-  private val form = formProvider()
   protected val controllerComponents: MessagesControllerComponents = cc
 
   def onPageLoad(mode: Mode, period: Period, index: Index): Action[AnyContent] = cc.authAndGetData(period) {
     implicit request =>
 
+      val form = formProvider(index, request.userAnswers.get(AllSalesFromNiQuery).getOrElse(Seq.empty).map(_.countryOfConsumption))
```

Comment:
Can we space this over multiple lines and the form in the onSubmit

Reply:
Done